kernel-install/50-mkosi.install

```diff
@@ -129,65 +141,19 @@ def main() -> None:
     output = "initrd" if format == OutputFormat.cpio else "uki"
 
     cmdline: list[PathString] = [
-        "mkosi",
-        "--directory", "",
+        "mkosi-initrd",
+        "--kernel-version", context.kernel_version,
         "--format", str(format),
         "--output", output,
-        "--workspace-dir=/var/tmp",
-        "--package-cache-dir=/var",
-        "--cache-only=metadata",
         "--output-dir", context.staging_area,
-        "--extra-tree", f"/usr/lib/modules/{context.kernel_version}:/usr/lib/modules/{context.kernel_version}",
-        "--extra-tree=/usr/lib/firmware:/usr/lib/firmware",
-        "--kernel-modules-exclude=.*",
-        "--kernel-modules-include=host",
-        "--include=mkosi-initrd",
     ]
 
     if context.verbose:
         cmdline += ["--debug"]
```

Comment:
This is still missing from mkosi-initrd which means it'll fail when running kernel-install in verbose mode. We should add a --debug argument to mkosi-initrd as well.